### PR TITLE
Website with database connected

### PR DESCRIPTION
**Pull request by wintersanchez**
connected database to php and displayed database on website with css.
added delete function from story 2 but not relevant yet.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_store
```
Comment (norayna) [inline .gitignore]: need to add `.idea` folder to `.gitignore` and delete the folder from the repo (currently it is in and it has 4 files in it - they all have to be removed from the repo)

File: cats_view.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"id","name","colour","weight","age","breed"
+1,"Nacho","brown",5.1,8,"Balinese"
+2,"Rain","black",3.7,4,"Bombay"
+3,"Bunny","white",3.5,3,"Angora"
+4,"Flapjack","red",6.2,12,"Maine Coon"
+5,"Fluffy","orange",2.8,9,"Burmese"
+6,"Daisy","black",4.9,7,"Nebelung"
```
Comment (norayna) [inline cats_view.csv:7]: what is this file?

File: index.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    echo "<br>";
+    echo "<form method='POST'>";
+    echo "<input type='hidden' name='delete' value='" . $row["id"] . "' />";
+    echo "<button type='submit' class='deleteButton' alt='delete-item-button'>Delete</button>";
+    echo "</form>";
+    echo "</div>";
+}
+echo "</section>";
+
+?>
```
Comment (norayna) [inline index.php]: Need to put all of this into a function in a separate file called `functions.php` and require in this file at the top before all the db stuff. The function should not use echo, because that is not testable, but you can return a string from the function and echo the result of the function in here.

File: index.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    $deleteCatInfo -> execute();
+}
+
+$getCatInfo = $db -> prepare("SELECT * FROM cats");
+
+$getCatInfo -> execute();
+
+$allCatInfo = $getCatInfo->fetchAll(\PDO::FETCH_ASSOC);
+
+$catItemNumber = 1;
```
Comment (norayna) [inline index.php:17]: Need to move all of this stuff at the top of the file (this is where it usually goes). Something like:
```php
<?php

// db stuff
?>
<html>
...
<?php
echo the cats
?>
...
</html>
```

File: new-collector-page-test/winter-collector-page-test.php [hunk 5]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (norayna) [inline new-collector-page-test/winter-collector-page-test.php:1]: Need to write unit tests for the function that you will create to generate the HTML of all the cats.

File: winter-collector-page.css [hunk 6]
```diff
@@ -0,0 +1,207 @@
+html {
```
Comment (norayna) [inline winter-collector-page.css:1]: I can't review the CSS until I get the db which isn't here currently

**Review by norayna — COMMENTED** [review]
The sql file is empty - you need to get it by exporting from Sequel Pro and replacing the file. Also other comments.

File: cats_view.csv (hunk 2 above)
Comment (norayna) [reply cats_view.csv:7]: remove from the repo

File: index.php [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+</head>
+<body>
+
+
+<nav class="navBar" alt="navigation-bar">
+    <ul class="navBarItems" alt="navigation-bar-links">
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="home">Home</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="your-profile">Your Profile</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="add-new-cats">Add New Cats?</a></li>
+        <li class="navBarItem"><a class="navBarLink" href="#" alt="contact">Contact</a></li>
```
Comment (norayna) [inline index.php]: it's not really advisable to have dead links, I can see you've put them there to make the page look more "real" but as a Product Owner I didn't ask for them. The master/main branch is supposed to only have finished, presentable functionality that can potentially be published online and be usable.

Tldr these links need to be removed (including the Add New Cats one because that's story 2)

File: winter-collector-page.css [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+ul {
+    list-style-type:none;
+    background: rgb(168, 149, 123);
+    margin: 15px;
+    border-radius: 5px;
+}
+
+div {
+    width: 30%;
+}
```
Comment (norayna) [inline winter-collector-page.css]: best to give a class to that div, otherwise in future if you add any other div tags they will be affected by this rule and you will have to change it

**Review by norayna — CHANGES_REQUESTED** [review]
more comments

File: winter-collector-page.css (hunk 8 above)
Comment (wintersanchez) [reply winter-collector-page.css]: removed the div -- wasn't needed anymore anyway! : ]
